### ISSUE TITLE
startTransition doesn't defer rendering of Suspense fallback

### ISSUE THREAD

**Issue by ellemedit** [facebook/react-native#42101]
### Description

In React(DOM), `startTransition` yield rendering till required data prepared to next paint if its trigger is inside Suspense boundary.

In latest ReactNative, I see `startTransition` doesn't yield rendering and immediately render nearest parent Suspense fallback even if required data isn't ready.

I didn't check previous releases.

### Steps to reproduce

```
cd ios
bundle install && RCT_NEW_ARCH_ENABLED=1 bundle exec pod install
cd ..
yarn ios
```

```
yarn android
```

On both two platforms:

1. By clicking `urgent` button, it renders suspense fallback. It's expected.
2. By clicking `transition` button, it renders suspense fallback. It's not expected.

### React Native Version

0.73.1

### Affected Platforms

Runtime - Android, Runtime - iOS

### Areas

Fabric - The New Renderer

### Output of `npx react-native info`

```text
System:
  OS: macOS 14.2.1
  CPU: (10) arm64 Apple M1 Pro
  Memory: 408.05 MB / 32.00 GB
  Shell:
    version: "5.9"
    path: /bin/zsh
Binaries:
  Node:
    version: 18.15.0
    path: ~/.nvm/versions/node/v18.15.0/bin/node
  Yarn:
    version: 1.22.19
    path: ~/.nvm/versions/node/v18.15.0/bin/yarn
  npm:
    version: 9.6.3
    path: ~/.nvm/versions/node/v18.15.0/bin/npm
  Watchman:
    version: 2023.12.04.00
    path: /opt/homebrew/bin/watchman
Managers:
  CocoaPods:
    version: 1.14.3
    path: /Users/bahwanhan/.rbenv/shims/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 23.2
      - iOS 17.2
      - macOS 14.2
      - tvOS 17.2
      - watchOS 10.2
  Android SDK: Not Found
IDEs:
  Android Studio: 2022.3 AI-223.8836.35.2231.11005911
  Xcode:
    version: 15.1/15C65
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 17.0.3
    path: /Users/bahwanhan/.sdkman/candidates/java/current/bin/javac
  Ruby:
    version: 2.6.10
    path: /Users/bahwanhan/.rbenv/shims/ruby
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.73.1
    wanted: 0.73.1
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: true
iOS:
  hermesEnabled: true
  newArchEnabled: true
```
```


### Stacktrace or Logs

```text
Last login: Sat Dec 30 13:17:06 on ttys006
/Users/bahwanhan/dev/rn-transition-not-work/ReproducerApp/node_modules/.bin/launchPackager.command ; exit;
➜  ~ /Users/bahwanhan/dev/rn-transition-not-work/ReproducerApp/node_modules/.bin/launchPackager.command ; exit;
info Welcome to React Native v0.73
info Starting dev server on port 8081...

                        ▒▒▓▓▓▓▒▒
                     ▒▓▓▓▒▒░░▒▒▓▓▓▒
                  ▒▓▓▓▓░░░▒▒▒▒░░░▓▓▓▓▒
                 ▓▓▒▒▒▓▓▓▓▓▓▓▓▓▓▓▓▒▒▒▓▓
                 ▓▓░░░░░▒▓▓▓▓▓▓▒░░░░░▓▓
                 ▓▓░░▓▓▒░░░▒▒░░░▒▓▒░░▓▓
                 ▓▓░░▓▓▓▓▓▒▒▒▒▓▓▓▓▒░░▓▓
                 ▓▓░░▓▓▓▓▓▓▓▓▓▓▓▓▓▒░░▓▓
                 ▓▓▒░░▒▒▓▓▓▓▓▓▓▓▒░░░▒▓▓
                  ▒▓▓▓▒░░░▒▓▓▒░░░▒▓▓▓▒
                     ▒▓▓▓▒░░░░▒▓▓▓▒
                        ▒▒▓▓▓▓▒▒


                Welcome to Metro v0.80.2
              Fast - Scalable - Integrated


warning: Watchman `watch-project` returned a warning: Recrawled this watch 1 times, most recently because:
MustScanSubDirs UserDroppedTo resolve, please review the information on
https://facebook.github.io/watchman/docs/troubleshooting.html#recrawl
To clear this warning, run:
`watchman watch-del '/Users/bahwanhan/dev/rn-transition-not-work/ReproducerApp' ; watchman watch-project '/Users/bahwanhan/dev/rn-transition-not-work/ReproducerApp'`

warning: Watchman `query` returned a warning: Recrawled this watch 1 times, most recently because:
MustScanSubDirs UserDroppedTo resolve, please review the information on
https://facebook.github.io/watchman/docs/troubleshooting.html#recrawl
To clear this warning, run:
`watchman watch-del '/Users/bahwanhan/dev/rn-transition-not-work/ReproducerApp' ; watchman watch-project '/Users/bahwanhan/dev/rn-transition-not-work/ReproducerApp'`

Recrawled this watch 1 times, most recently because:
MustScanSubDirs UserDroppedTo resolve, please review the information on
https://facebook.github.io/watchman/docs/troubleshooting.html#recrawl
To clear this warning, run:
`watchman watch-del '/Users/bahwanhan/dev/rn-transition-not-work/ReproducerApp' ; watchman watch-project '/Users/bahwanhan/dev/rn-transition-not-work/ReproducerApp'`

info Dev server ready

i - run on iOS
a - run on Android
d - open Dev Menu
r - reload app

info Opening app on iOS...
info A dev server is already running for this project on port 8081.
info Found Xcode workspace "ReproducerApp.xcworkspace"
info Found booted iPhone SE (3rd generation)
info Building (using "xcodebuild -workspace ReproducerApp.xcworkspace -configuration Debug -scheme ReproducerApp -destination id=3277291D-01B9-448E-BE2D-659D2ADE7B0B")

info 💡 Tip: Make sure that you have set up your development environment correctly, by running npx react-native doctor. To read more about doctor command visit: https://github.com/react-native-community/cli/blob/main/packages/cli-doctor/README.md#doctor 

Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -workspace ReproducerApp.xcworkspace -configuration Debug -scheme ReproducerApp -destination id=3277291D-01B9-448E-BE2D-659D2ADE7B0B

User defaults from command line:
    IDEPackageSupportUseBuiltinSCM = YES

Prepare packages

ComputeTargetDependencyGraph
note: Building targets in dependency order
note: Target dependency graph (67 targets)
    Target 'ReproducerApp' in project 'ReproducerApp'
        ➜ Implicit dependency on target 'Pods-ReproducerApp' in project 'Pods' via file 'libPods-ReproducerApp.a' in build phase 'Link Binary'
        ➜ Implicit dependency on target 'CocoaAsyncSocket' in project 'Pods' via options '-lCocoaAsyncSocket' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'DoubleConversion' in project 'Pods' via options '-lDoubleConversion' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'Flipper' in project 'Pods' via options '-lFlipper' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'Flipper-Boost-iOSX' in project 'Pods' via options '-lFlipper-Boost-iOSX' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'Flipper-Fmt' in project 'Pods' via options '-lFlipper-Fmt' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'Flipper-Folly' in project 'Pods' via options '-lFlipper-Folly' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'Flipper-PeerTalk' in project 'Pods' via options '-lFlipper-PeerTalk' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'FlipperKit' in project 'Pods' via options '-lFlipperKit' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'RCT-Folly' in project 'Pods' via options '-lRCT-Folly' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'RCTTypeSafety' in project 'Pods' via options '-lRCTTypeSafety' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-Codegen' in project 'Pods' via options '-lReact-Codegen' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-Core' in project 'Pods' via options '-lReact-Core' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-CoreModules' in project 'Pods' via options '-lReact-CoreModules' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-Fabric' in project 'Pods' via options '-lReact-Fabric' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-FabricImage' in project 'Pods' via options '-lReact-FabricImage' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-ImageManager' in project 'Pods' via options '-lReact-ImageManager' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-Mapbuffer' in project 'Pods' via options '-lReact-Mapbuffer' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-NativeModulesApple' in project 'Pods' via options '-lReact-NativeModulesApple' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RCTAnimation' in project 'Pods' via options '-lReact-RCTAnimation' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RCTAppDelegate' in project 'Pods' via options '-lReact-RCTAppDelegate' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RCTBlob' in project 'Pods' via options '-lReact-RCTBlob' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RCTFabric' in project 'Pods' via options '-lReact-RCTFabric' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RCTImage' in project 'Pods' via options '-lReact-RCTImage' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RCTLinking' in project 'Pods' via options '-lReact-RCTLinking' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RCTNetwork' in project 'Pods' via options '-lReact-RCTNetwork' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RCTSettings' in project 'Pods' via options '-lReact-RCTSettings' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RCTText' in project 'Pods' via options '-lReact-RCTText' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RCTVibration' in project 'Pods' via options '-lReact-RCTVibration' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RuntimeApple' in project 'Pods' via options '-lReact-RuntimeApple' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RuntimeCore' in project 'Pods' via options '-lReact-RuntimeCore' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-RuntimeHermes' in project 'Pods' via options '-lReact-RuntimeHermes' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-cxxreact' in project 'Pods' via options '-lReact-cxxreact' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-debug' in project 'Pods' via options '-lReact-debug' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-graphics' in project 'Pods' via options '-lReact-graphics' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-hermes' in project 'Pods' via options '-lReact-hermes' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-jserrorhandler' in project 'Pods' via options '-lReact-jserrorhandler' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-jsi' in project 'Pods' via options '-lReact-jsi' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-jsiexecutor' in project 'Pods' via options '-lReact-jsiexecutor' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-jsinspector' in project 'Pods' via options '-lReact-jsinspector' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-jsitracing' in project 'Pods' via options '-lReact-jsitracing' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-logger' in project 'Pods' via options '-lReact-logger' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-nativeconfig' in project 'Pods' via options '-lReact-nativeconfig' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-perflogger' in project 'Pods' via options '-lReact-perflogger' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-rendererdebug' in project 'Pods' via options '-lReact-rendererdebug' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-runtimescheduler' in project 'Pods' via options '-lReact-runtimescheduler' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'React-utils' in project 'Pods' via options '-lReact-utils' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'ReactCommon' in project 'Pods' via options '-lReactCommon' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'SocketRocket' in project 'Pods' via options '-lSocketRocket' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'Yoga' in project 'Pods' via options '-lYoga' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'fmt' in project 'Pods' via options '-lfmt' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'glog' in project 'Pods' via options '-lglog' in build setting 'OTHER_LDFLAGS'
        ➜ Implicit dependency on target 'libevent' in project 'Pods' via options '-llibevent' in build setting 'OTHER_LDFLAGS'
    Target 'Pods-ReproducerApp' in project 'Pods'
        ➜ Explicit dependency on target 'CocoaAsyncSocket' in project 'Pods'
        ➜ Explicit dependency on target 'DoubleConversion' in project 'Pods'
        ➜ Explicit dependency on target 'FBLazyVector' in project 'Pods'
        ➜ Explicit dependency on target 'Flipper' in project 'Pods'
        ➜ Explicit dependency on target 'Flipper-Boost-iOSX' in project 'Pods'
        ➜ Explicit dependency on target 'Flipper-DoubleConversion' in project 'Pods'
        ➜ Explicit dependency on target 'Flipper-Fmt' in project 'Pods'
        ➜ Explicit dependency on target 'Flipper-Folly' in project 'Pods'
        ➜ Explicit dependency on target 'Flipper-Glog' in project 'Pods'
        ➜ Explicit dependency on target 'Flipper-PeerTalk' in project 'Pods'
        ➜ Explicit dependency on target 'FlipperKit' in project 'Pods'
        ➜ Explicit dependency on target 'OpenSSL-Universal' in project 'Pods'
        ➜ Explicit dependency on target 'RCT-Folly' in project 'Pods'
        ➜ Explicit dependency on target 'RCTRequired' in project 'Pods'
        ➜ Explicit dependency on target 'RCTTypeSafety' in project 'Pods'
        ➜ Explicit dependency on target 'React' in project 'Pods'
        ➜ Explicit dependency on target 'React-Codegen' in project 'Pods'
        ➜ Explicit dependency on target 'React-Core' in project 'Pods'
        ➜ Explicit dependency on target 'React-CoreModules' in project 'Pods'
        ➜ Explicit dependency on target 'React-Fabric' in project 'Pods'
        ➜ Explicit dependency on target 'React-FabricImage' in project 'Pods'
        ➜ Explicit dependency on target 'React-ImageManager' in project 'Pods'
        ➜ Explicit dependency on target 'React-Mapbuffer' in project 'Pods'
        ➜ Explicit dependency on target 'React-NativeModulesApple' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTActionSheet' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTAnimation' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTAppDelegate' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTBlob' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTFabric' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTImage' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTLinking' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTNetwork' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTSettings' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTText' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTVibration' in project 'Pods'
        ➜ Explicit dependency on target 'React-RuntimeApple' in project 'Pods'
        ➜ Explicit dependency on target 'React-RuntimeCore' in project 'Pods'
        ➜ Explicit dependency on target 'React-RuntimeHermes' in project 'Pods'
        ➜ Explicit dependency on target 'React-callinvoker' in project 'Pods'
        ➜ Explicit dependency on target 'React-cxxreact' in project 'Pods'
        ➜ Explicit dependency on target 'React-debug' in project 'Pods'
        ➜ Explicit dependency on target 'React-graphics' in project 'Pods'
        ➜ Explicit dependency on target 'React-hermes' in project 'Pods'
        ➜ Explicit dependency on target 'React-jserrorhandler' in project 'Pods'
        ➜ Explicit dependency on target 'React-jsi' in project 'Pods'
        ➜ Explicit dependency on target 'React-jsiexecutor' in project 'Pods'
        ➜ Explicit dependency on target 'React-jsinspector' in project 'Pods'
        ➜ Explicit dependency on target 'React-jsitracing' in project 'Pods'
        ➜ Explicit dependency on target 'React-logger' in project 'Pods'
        ➜ Explicit dependency on target 'React-nativeconfig' in project 'Pods'
        ➜ Explicit dependency on target 'React-perflogger' in project 'Pods'
        ➜ Explicit dependency on target 'React-rendererdebug' in project 'Pods'
        ➜ Explicit dependency on target 'React-rncore' in project 'Pods'
        ➜ Explicit dependency on target 'React-runtimeexecutor' in project 'Pods'
        ➜ Explicit dependency on target 'React-runtimescheduler' in project 'Pods'
        ➜ Explicit dependency on target 'React-utils' in project 'Pods'
        ➜ Explicit dependency on target 'ReactCommon' in project 'Pods'
        ➜ Explicit dependency on target 'SocketRocket' in project 'Pods'
        ➜ Explicit dependency on target 'Yoga' in project 'Pods'
        ➜ Explicit dependency on target 'boost' in project 'Pods'
        ➜ Explicit dependency on target 'fmt' in project 'Pods'
        ➜ Explicit dependency on target 'glog' in project 'Pods'
        ➜ Explicit dependency on target 'hermes-engine' in project 'Pods'
        ➜ Explicit dependency on target 'libevent' in project 'Pods'
    Target 'React-rncore' in project 'Pods' (no dependencies)
    Target 'React-RCTAppDelegate' in project 'Pods'
        ➜ Explicit dependency on target 'RCT-Folly' in project 'Pods'
        ➜ Explicit dependency on target 'RCTRequired' in project 'Pods'
        ➜ Explicit dependency on target 'RCTTypeSafety' in project 'Pods'
        ➜ Explicit dependency on target 'React-Core' in project 'Pods'
        ➜ Explicit dependency on target 'React-CoreModules' in project 'Pods'
        ➜ Explicit dependency on target 'React-Fabric' in project 'Pods'
        ➜ Explicit dependency on target 'React-NativeModulesApple' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTFabric' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTImage' in project 'Pods'
        ➜ Explicit dependency on target 'React-RCTNetwork' in project 'Pods'
        ➜ Explicit dependency on target 'React-RuntimeApple' in project 'Pods'
        ➜ Explicit dependency on target 'React-RuntimeCore' in project 'Pods'
        ➜ Explicit dependency on target 'React-RuntimeHermes' in project 'Pods'
        ➜ Explicit dependency on target 'React-debug' in project 'Pods'
        ➜ Explicit dependency on target 'React-graphics' in project 'Pods'
        ➜ Explicit dependency on target 'React-hermes' in project 'Pods'
        ➜ Explicit dependency on target 'React-nativeconfig' in project 'Pods'
        ➜ Explicit dependency on target 'React-rendererdebug' in project 'Pods'
        ➜ Explicit dependency on target 'React-runtimescheduler' in project 'Pods'
        ➜ Explicit dependency on target 'React-utils' in project 'Pods'
        ➜ Explicit dependency on target 'ReactCommon' in project 'Pods'
    Target 'React-RuntimeApple' in project 'Pods'
        ➜ Explicit dependency on target 'RCT-Folly' in project 'Pods'
        ➜ Explicit dependency on target 'React-Core' in project 'Pods'
        ➜ Explicit dependency on target 'React-CoreModules' in project 'Pods'
        ➜ Explicit dependency on target 'React-Mapbuffer' in project 'Pods'
 BUNDLE  ./index.js 

 LOG  Running "ReproducerApp" with {"rootTag":1,"initialProps":{"concurrentRoot":true},"fabric":true}
```

### Reproducer

https://github.com/ellemedit/rn-transition-bug-reproduce

### Screenshots and Videos

https://github.com/facebook/react-native/assets/2682865/365b82b8-95c7-4398-a20b-c9215fd15c5b

**Comment by dlindenkreuz** [comment]
Still reproducible with RN 0.74.5.

**Comment by cipolleschi** [comment]
@ellemedit @dlindenkreuz Thanks for the report. This is known issue internally and we already have a fix almost ready. We are running an experiment on that to make sure it does not have unexpected side effects, but we will probably try to land it for 0.76

**Comment by budde377** [comment]
Is there any update on this issue, @cipollesch?

**Comment by yuliswe** [comment]
Still happening for rn 0.76.5

**Comment by yuliswe** [comment]
Related discussion
https://github.com/reactwg/react-native-new-architecture/discussions/94

**Comment by cipolleschi** [comment]
Hi everyone! sorry for the delay. We had a fix for this and we were experimenting with it internally.
Unfortunately, the fix was causing some regressions on internal metrics and we are not able to ship it right now.

Starting from 0.78, React Native will ship with React 19. We expect for this issue to be solved by React 19. We verified that this issue does not reproduce when using `use`, that is the expected way to use `Suspense`.

**Comment by dlindenkreuz** [comment]
Still reproducible with RN 0.79.2 and Expo SDK 53: https://github.com/dlindenkreuz/rn-19-transition-deferred-bug

Neither `React.useDeferredValue` nor `React.useTransition` work as expected. The repro app shows a Suspense fallback.

I assume my dummy Promise cache is not the issue…

**Comment by cipolleschi** [comment]
Hi @dlindenkreuz thanks for re-surfacing the issue. I'll reopen the issue and hope to look into this soon.

**Comment by cipolleschi** [comment]
We merged React 19.1.1 in main, and we verified that it fixes the issue.
The bug will be fixed in React Native 0.82

**Comment by dlindenkreuz** [comment]
@cipolleschi Is it possible to cherry-pick the fix into a 0.81.x patch? Suspense doesn't work properly without this fix and there are no known workarounds 😭

**Comment by daannet** [comment]
@dlindenkreuz  I'm using 0.82 rc3 at the moment, with Expo canary
